README.md

```diff
@@ -1,6 +1,6 @@
 [![CircleCI](https://circleci.com/gh/pytorch/xla.svg?style=svg)](https://circleci.com/gh/pytorch/xla)
 
-# How to Run PyTorch with TPUs
+# How to Run PyTorch with single TPU nodes
```

Comment:
`single TPU nodes` -> `Single TPU Nodes`

Reply:
I'd also try to see if we can squeeze in something like the link below (unless we have it elsewhere), as when I first read this, I thought Single Core:

https://cloud.google.com/tpu/docs/system-architecture